src/components/forms/ObjectStorageForm.js

```diff
@@ -74,7 +84,7 @@ class ObjectStorageInstancesForm extends Component {
             invalidText={this.props.invalidTextCallback(this.state, this.props)}
           />
           <IcseSelect
-            formName="resource_group"
+            formName="object_storage"
```

Comment:
formname is used to create the id, to ensure uniqueness let's pass `this.props.data.name` in addition to `object-storage-rg`. let's also ensure the rest of the selects follow this pattern